lib/resources/example_json/r4_examples_charge_item.rb

```diff
@@ -0,0 +1,149 @@
+# frozen_string_literal: true
+
+module Cerner
+  module Resources
+
+    R4_CHARGE_ITEM_READ ||= {
+      'resourceType': 'ChargeItem',
+      'id': '157320807',
+      'meta': {
+        'versionId': '3',
+        'lastUpdated': '2019-08-15T14:22:31Z'
+      },
+      'text': {
+        'status': 'generated',
+        'div': '<div xmlns="http://www.w3.org/1999/xhtml"><p><b>Charge Item</b></p><p><b>Status</b>: Billable</p>' \
+               '<p><b>Patient</b>: 13393174</p><p><b>Encounter</b>: 11945114</p><p><b>Code</b>: 34523</p>' \
+               '<p><b>Occurrence</b>: Aug  4, 2019  4:38 A.M. CDT</p></div>'
+      },
+      'extension': [
+        {
+          'valueMoney': {
+            'value': 15,
+            'currency': 'USD'
+          },
+          'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/unit-price'
+        },
+        {
+          'valueMoney': {
+            'value': 15,
+            'currency': 'USD'
+          },
+          'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/net-price'
+        },
+        {
+          'valueReference': {
+            'reference': 'ChargeItem/157320880'
+          },
+          'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/offset-by'
+        },
+        {
+          'valueString': 'CDM Description',
+          'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/description'
+        },
+        {
+          'extension': [
+            {
+              'valueCodeableConcept': {
+                'coding': [
+                  {
+                    'system': 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/CodeSystem/BillCodes-HCPCS',
+                    'code': '43'
+                  }
+                ]
+              },
+              'url': 'code'
+            },
+            {
+              'valueUnsignedInteger': 1,
```

Comment:
this should be valueUnsignedInt.
https://www.hl7.org/fhir/extensibility.html

Reply:
updated here: https://github.com/cerner/fhir.cerner.com/pull/605/commits/814ca55f3733893c385dca50927276addf1e99ae#diff-5cd46ad5f0af8a24217f67b656e77ce3624d6cefec3a18eddf7adc12bd47d35fR58